requirements-dev.txt

```diff
@@ -1,6 +1,8 @@
-pytest>=7.2.0
```

Comment:
Why was pytest removed?

Reply:
It is unused.

Reply:
The Makefile still has some hooks for it but there are not any actual tests.

Reply:
removed~